axon-server/performance/flow-control.md

```diff
@@ -29,5 +29,26 @@ Set the following properties to set flow control on the synchronization between
 * `axoniq.axonserver.queryFlowControl.nr-of-new-permits` \[5000\] - additional number of messages that the master can send to replica.
 * `axoniq.axonserver.queryFlowControl.new-permits-threshold` \[5000\] - when replica reaches this threshold in remaining messages, it sends a request with additional number of messages to receive.
 
+## Streaming query
```

Comment:
The streaming query stuff is for 4.6.0, not 4.5.
As such, it doesn't belong in this PR.

@schananas added these to master since master will become the 4.6.0. branch.
I guess you've updated 4.5 with master though.

Reply:
Ok, sorry about that. Can I easily remove those bits? Otherwise, we'll just merge this PR into master rather than 4.5. and copy over these two files to 4.5.

Reply:
Hmm. If we remove them here, we'll eventually overwrite them from master once 4.5 is merged in to master (to ensure the upcoming 4.6.0 Refguide has your text).
Hence a commit that removes wouldn't work I'm afraid.

Can't you revert the merge with `master` from this branch?
Assuming that's what you've done, of course.

Reply:
Let's keep it simple: This PR concerns only two files, the rest is from `master`. I'll copy them, delete branch and PR, and redo everything.

Reply:
Ah, that definitely sounds a lot simpler. I'd say go for it :+1: